### DISCUSSION TITLE
How to load tensorflow model in onnx java runtime and make inference

### DISCUSSION THREAD

**Discussion by bejugamvarun** [microsoft/onnxruntime#5144] [Other Q&A]
I created a simple regression model in tensorflow and converted it into the onnx format using the converter and i couldn't load my onnx model in java runtime i used the maven dependency provided and I couldn't find any clear documentation regarding the java api for onnx runtime

**Comment by Craigacp** [comment]
What steps did you try? There are many examples of loading models in the Java unit tests.
